.circleci/config.yml

```diff
@@ -54,10 +54,19 @@ jobs:
           docker login -u $DOCKER_USER -p $DOCKER_PASS
       - run: |
           docker build -t envirodgi/processing:$CIRCLE_SHA1 .
+          docker build -t envirodgi/ia-healthcheck:$CIRCLE_SHA1 -f Dockerfile.ia-healthcheck .
+          docker build -t envirodgi/ia-etl:$CIRCLE_SHA1 -f Dockerfile.ia-etl .
           docker build -t envirodgi/processing:latest .
+          docker build -t envirodgi/ia-healthcheck:latest -f Dockerfile.ia-healthcheck .
+          docker build -t envirodgi/ia-etl:latest -f Dockerfile.ia-etl .
```

Comment:
It’s a little more efficient to use `docker tag <source> <target>` here. Not sure why we weren’t doing that before. e.g:

```
docker tag envirodgi/ia-etl:$CIRCLE_SHA1 envirodgi/ia-etl:latest
```